### PR TITLE
Adds option to exclude channels from classifier

### PR DESCRIPTION
**Pull request by zduey**
Adds another argument to CLI for config generation and report building that allows users to exclude labels by populating a file called classifier_excluded_leads.txt in the tal/ directory

### CODE REVIEW

File: ramutils/cli/expconf.py [hunk 1]
```diff
@@ -182,62 +182,11 @@ def create_expconf(input_args=None):
...
+                              use_common_reference=args.use_common_reference,
+                              use_classifier_excluded_leads=args.use_classifier_excluded_leads)
         memory.clear()  # clear cached intermediate results on successful build
 
     warnings = '\n' + warning_accumulator.format_all()
     if warnings is not None:
         logger.info(warnings)
 
-
-if __name__ == "__main__":
```
Comment (mivade) [inline ramutils/cli/expconf.py]: You just *have* to make my life harder, don't you?

File: ramutils/montage.py [hunk 2]
```diff
@@ -542,6 +548,39 @@ def get_pairs(subject, experiment, sessions, paths):
     return all_pairs
 
 
+def get_classifier_excluded_leads(subject, rootdir='/'):
```
Comment (mivade) [inline ramutils/montage.py]: Is `rootdir` the usual parameter name for this?

Comment (zduey) [reply ramutils/montage.py]: Yeah. I'm not sure that I am 100% consistent, but I've been trying to keep it rootdir. The other obvious options are rhino_root which I know I dont use and just root

File: ramutils/cli/expconf.py (hunk 1 above)
Comment (zduey) [reply ramutils/cli/expconf.py]: That's what the demos are for!

Comment (mivade) [reply ramutils/cli/expconf.py]: I use that section when debugging issues found in the wild, though.

Comment (mivade) [reply ramutils/cli/expconf.py]: If you're concerned about coverage reporting, you can put `# pragma: nocover` at the end of line 193 or configure coverage to ignore all `if __name__ == "__main__"` blocks.

**Comment by zduey** [comment]
@mivade  -- Added your in the wild debugging option back to config generation

**Pull request merged**